<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KTransactionFilter</class>
 <widget class="QWidget" name="KTransactionFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search transactions</string>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="TextLabel1">
     <property name="text">
      <string>Define your search criteria</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="m_criteriaTab">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_textTab">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="3" column="2">
        <spacer name="Spacer20">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>16</width>
           <height>83</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="KLineEdit" name="m_textEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="showClearButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="KComboBox" name="m_textNegate">
         <item>
          <property name="text">
           <string>Contains</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Does not contain</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QCheckBox" name="m_regExp">
         <property name="text">
          <string>Treat text as regular expression</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="m_caseSensitive">
         <property name="text">
          <string>Case sensitive</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_accountTab">
      <attribute name="title">
       <string>Account</string>
      </attribute>
      <layout class="QVBoxLayout" name="_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>11</number>
       </property>
       <item>
        <widget class="KMyMoneyAccountSelector" name="m_accountsView" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_dateTab">
      <attribute name="title">
       <string>Date</string>
      </attribute>
      <layout class="QVBoxLayout" name="dateTabLayout">
       <item>
        <layout class="QHBoxLayout" name="dateRangeLayout">
         <item>
          <spacer name="Spacer25">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="Spacer22">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_amountTab">
      <attribute name="title">
       <string>Amount</string>
      </attribute>
      <layout class="QVBoxLayout" name="_3">
       <item>
        <layout class="QHBoxLayout" name="_4">
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <layout class="QVBoxLayout" name="_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="m_amountButton">
              <property name="text">
               <string>Search this amount</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_amountRangeButton">
              <property name="text">
               <string>Search amount in the range</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="_6">
           <item row="2" column="1">
            <widget class="AmountEdit" name="m_amountToEdit" native="true">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="TextLabel9">
             <property name="text">
              <string/>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="TextLabel8">
             <property name="text">
              <string comment="to (amount)">to</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="AmountEdit" name="m_amountEdit" native="true">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="TextLabel7">
             <property name="text">
              <string comment="from (amount)">from</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="AmountEdit" name="m_amountFromEdit" native="true">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="Spacer28">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="Spacer27">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>16</width>
           <height>75</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_categoryTab">
      <attribute name="title">
       <string>Category</string>
      </attribute>
      <layout class="QVBoxLayout" name="_7">
       <item>
        <widget class="KMyMoneyAccountSelector" name="m_categoriesView" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tagTab">
      <attribute name="title">
       <string>Tag</string>
      </attribute>
      <layout class="QHBoxLayout" name="_8">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>11</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="_9">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="m_tagsView">
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <column>
            <property name="text">
             <string>Tag</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="m_emptyTagsButton">
           <property name="text">
            <string>Select transactions without tags</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="_10">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="m_allTagsButton">
           <property name="text">
            <string>Select all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_clearTagsButton">
           <property name="text">
            <string>Select none</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="Spacer29">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_payeeTab">
      <attribute name="title">
       <string>Payee</string>
      </attribute>
      <layout class="QHBoxLayout" name="_11">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>11</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="_12">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="m_payeesView">
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <column>
            <property name="text">
             <string>Payee</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="m_emptyPayeesButton">
           <property name="text">
            <string>Select transactions without payees</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="_13">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="m_allPayeesButton">
           <property name="text">
            <string>Select all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_clearPayeesButton">
           <property name="text">
            <string>Select none</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="Spacer291">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_detailsTab">
      <attribute name="title">
       <string>Details</string>
      </attribute>
      <layout class="QVBoxLayout" name="_14">
       <item>
        <layout class="QHBoxLayout" name="_15">
         <item>
          <layout class="QGridLayout" name="_16">
           <item row="2" column="0">
            <widget class="QLabel" name="TextLabel11_2">
             <property name="text">
              <string>Validity</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="TextLabel10">
             <property name="text">
              <string>Type</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KComboBox" name="m_stateBox">
             <item>
              <property name="text">
               <string comment="@item reconciliation status">All states</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string comment="@item reconciliation status">Not reconciled</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string comment="@item reconciliation status">Cleared</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string comment="@item reconciliation status">Reconciled</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KComboBox" name="m_typeBox">
             <item>
              <property name="text">
               <string>All types</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Payments</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Deposits</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Transfers</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KComboBox" name="m_validityBox">
             <item>
              <property name="text">
               <string>Any transaction</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Valid transaction</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Invalid transaction</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="TextLabel11">
             <property name="text">
              <string comment="@label reconciliation status">State</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="Spacer30">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>130</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_17">
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <layout class="QVBoxLayout" name="_18">
            <item>
             <widget class="QRadioButton" name="m_nrButton">
              <property name="text">
               <string>Search this number</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_nrRangeButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Search number in range</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="_19">
           <item row="0" column="0">
            <widget class="QLabel" name="TextLabel12">
             <property name="text">
              <string/>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="TextLabel14">
             <property name="text">
              <string comment="to (number)">to</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KMyMoneyLineEdit" name="m_nrToEdit" native="true"/>
           </item>
           <item row="0" column="1">
            <widget class="KMyMoneyLineEdit" name="m_nrEdit" native="true"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="TextLabel13">
             <property name="text">
              <string comment="from (number)">from</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KMyMoneyLineEdit" name="m_nrFromEdit" native="true"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="Spacer31">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="Spacer32">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_selectedCriteria">
     <property name="text">
      <string>text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_transferWarning">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer16">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyLineEdit</class>
   <extends>QWidget</extends>
   <header>../widgets/kmymoneylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyAccountSelector</class>
   <extends>QWidget</extends>
   <header location="global">../widgets/kmymoneyaccountselector.h</header>
  </customwidget>
  <customwidget>                                   
   <class>AmountEdit</class>
   <extends>QLineEdit</extends>                   
   <header location="global">amountedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
